app/models/media_object.rb

```diff
@@ -67,6 +67,18 @@ class MediaObject < ActiveFedora::Base
   validates  :topical_subject, presence: true
   validate  :validate_access_restrictions_license
   validate  :validate_access_restrictions_rights
+  validate  :validate_genre
+
+  def validate_genre
+    genre_list = Array(genre)
+    if genre_list.empty?
+      errors.add(:genre, "At least one genre must be specified")
```

Comment:
Here you do 

```ruby
if X
   do_thing
end
```

But down a couple lines you use:

``` ruby
do_thing unless Y
```

How do you feel about normalizing it to use one-line constructs in both cases?